src/treemodel/node.js

```diff
@@ -188,6 +188,9 @@ export default class Node {
 		// Due to circular references we need to remove parent reference.
 		json.parent = this.parent ? this.parent.name : null;
 
+		// Serialize attributes as Map object is represented as "{}" when parsing to JSON.
+		json._attrs = [ ...json._attrs ];
```

Comment:
I think it is the bigger question: should serialised object keep private properties? I am not sure about the answers. It is simpler to use private properties, and this is what we do, for instance in `clone` or `isEqual` methods. On the other hand, serialised object, send over the network seems to be a public API. If it will use private properties then send objects will change every time private properties change, what looks like a bad smell.